pkg/http/server/server.go

```diff
@@ -159,18 +159,9 @@ func (s *Server) decodeAndStoreMetrics(ctx context.Context, partitionKey string,
 		return err
 	}
 
-	// filter the list
-	for i, family := range families {
```

Comment:
this was an exact re-write of the Filter transform, so let's substitute for conciseness.

Reply:
:+1: